/**:
  camera:
    ros__parameters:
      enable_color: true
      rgb_camera:
        profile: '640,480,60'
        enable_auto_exposure: false
      enable_depth: true
      enable_rgbd: true
      align_depth:
        enable: true
      pointcloud:
        enable: true
      enable_sync: true
      enable_infra: false
      enable_infra1: false
      enable_infra2: false

$(var cam_high_name):
  camera:
    ros__parameters:
      camera_name: "cam_high"
      serial_no: "130322272153"

$(var cam_low_name):
  camera:
    ros__parameters:
      camera_name: "cam_low"
      serial_no: ""

$(var cam_right_wrist_name):
  camera:
    ros__parameters:
      camera_name: "cam_right_wrist"
      serial_no: "130322270188"

$(var cam_left_wrist_name):
  camera:
    ros__parameters:
      camera_name: "cam_left_wrist"
      serial_no: "130322272267"
